debian/control

```diff
@@ -16,8 +16,8 @@ Build-Depends: cmake (>= 2.8.11),
                ninja-build,
                pandoc,
                pkg-config,
-               python3-docutils,
```

Comment:
We really need to put this wrap and sort in the CI.

Reply:
Feel free to do that. `wrap-and-sort` is part of the devscripts package: https://tracker.debian.org/pkg/devscripts